mod/src/main/kotlin/gg/skytils/skytilsmod/features/impl/slayer/SlayerFeatures.kt

```diff
@@ -256,7 +256,7 @@ object SlayerFeatures : EventSubscriber, CoroutineScope {
             val entity = event.entity as ArmorStandEntity
             if (!entity.hasCustomName()) return
             val name = entity.displayName?.string ?: return
-            if (Skytils.config.slayerBossHitbox && name.endsWith("§c❤") && !name.endsWith("§e0§c❤") && !mc.entityRenderDispatcher.shouldRenderHitboxes()) {
+            if (Skytils.config.slayerBossHitbox && name.endsWith("❤") && !name.endsWith(" 0❤") && !mc.entityRenderDispatcher.shouldRenderHitboxes()) {
```

Comment:
Are you sure this works? Assuming that name is still formatted, this will fail due to having no formatting codes